cmd/docker/docker.go

```diff
@@ -55,7 +55,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	cli.SetupRootCommand(cmd)
 
 	flags = cmd.Flags()
-	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
+	flags.BoolVarP(&opts.Version, "version", "v", false, "Show version information")
```

Comment:
No, `--version` is different from other flags such as `--config`, `--debug`, and `--tls`.
They does not quit.